questionpy_common/api/__init__.py

```diff
@@ -1,3 +1,23 @@
 #  This file is part of QuestionPy. (https://questionpy.org)
 #  QuestionPy is free software released under terms of the MIT license. See LICENSE.md.
 #  (c) Technische Universität Berlin, innoCampus <info@isis.tu-berlin.de>
+from collections.abc import (
+    Mapping,
+    Sequence,  # noqa: F401
+)
+from typing import (
+    TypeAlias,
+    Union,  # noqa: F401,
+)
+
+from pydantic import BaseModel
+from typing_extensions import TypeAliasType
+
+
+class Localized(BaseModel):
+    lang: str
+
+
+# "Regular" recursive type aliases break Pydantic: https://github.com/pydantic/pydantic/issues/8346
+PlainValue: TypeAlias = TypeAliasType("PlainValue", "Union[None, int, str, bool, Sequence[PlainValue], PlainMapping]")
```

Comment:
Ich denke, hier nutzt du nicht [`pydantic.JsonValue`](https://docs.pydantic.dev/2.8/api/types/#pydantic.types.JsonValue), weil die Pydantic-Definition invariant ist?
Aber fehlt in der `Union` für `PlainValue` nicht noch `float`?

Reply:
`pydantic.JsonValue` war mir einfach noch nicht bekannt. Ist tbf auch schwer zu finden, nach [pydantic.Json](https://docs.pydantic.dev/2.0/usage/types/json/) habe ich nicht lang weitergesucht. Ich glaube an der Stelle ist Kovarianz gar nicht nötig, daher ändere ich es :+1: